### PR TITLE
Billy

### PR DESCRIPTION
**Pull request by minhyeong-joe**

### CODE REVIEW

File: client/src/reducers/log_in_user_reducer.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function logInUserReducer(state = initialState, action) {
       return {
         ...state,
         data: "",
-        loggedIn: action.payload
+        loggedIn: ""
```
Comment (isaac1104) [inline client/src/reducers/log_in_user_reducer.js]: you need to return "action.payload"

**Review by isaac1104 — COMMENTED** [review]
log_in_user_reducer.js
line 26 see comment

File: client/src/reducers/log_in_user_reducer.js (hunk 1 above)
Comment (minhyeong-joe) [reply client/src/reducers/log_in_user_reducer.js]: For some reason, continue as guest was still looking for loggedin username, this is just a quick fix

Comment (isaac1104) [reply client/src/reducers/log_in_user_reducer.js]: well can you fix it back with using action.payload?

**Pull request closed without merging**